apps/examples/wifi_manager_sample/Kconfig

```diff
@@ -10,6 +10,78 @@ config EXAMPLES_WIFIMANAGER_TEST
 	---help---
 		Wi-Fi Manager sample program
 
+
 config USER_ENTRYPOINT
 	string
 	default "wm_test_main" if ENTRY_WIFIMANAGER_TEST
+
+
+if EXAMPLES_WIFIMANAGER_TEST
```

Comment:
I think 'EXAMPLES_WIFIMANGER_STRESS_TEST' is more proper word than 'EXAMPLES_WIFIMANGER_TEST'

Reply:
Ok I'll fix